foodsaving/userauth/api.py

```diff
@@ -100,7 +98,7 @@ def post(self, request):
         """
         Send token to verify e-mail
 
-        requires "key" parameter
+        requires "code" parameter
```

Comment:
Serializer still expects the `key` parameter, I would probably leave it that way for now.